features/step_definitions/assertions_steps.rb

```diff
@@ -2,10 +2,15 @@
     expect(page).to have_content content
 end
 
-Then("I should be on the landing page") do
-    expect(current_path).to eq root_path 
+Then("I should be on the {string} page") do |page|
```

Comment:
dont think we should use page as argument

Reply:
done